fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/catalog/FlinkCatalogITCase.java

```diff
@@ -421,6 +424,69 @@ void testCreateTableWithUnknownOptions() {
                 .hasMessageContaining("Unsupported options found for 'fluss'");
     }
 
+    @Test
+    void testAuthentication() throws Exception {
+        String clientListenerName = "CLIENT";
+        Configuration serverConfig = new Configuration();
+        serverConfig.setString(
+                ConfigOptions.SERVER_SECURITY_PROTOCOL_MAP.key(), "CLIENT:username_password");
+        serverConfig.setString("security.username_password.username", "root");
+        serverConfig.setString("security.username_password.password", "password");
+        FlussClusterExtension flussClusterExtension =
+                FlussClusterExtension.builder()
+                        .setCoordinatorServerListeners(
+                                String.format(
+                                        "%s://localhost:0, %s://localhost:0",
+                                        DEFAULT_LISTENER_NAME, clientListenerName))
+                        .setTabletServerListeners(
+                                String.format(
+                                        "%s://localhost:0, %s://localhost:0",
+                                        DEFAULT_LISTENER_NAME, clientListenerName))
+                        .setClusterConf(serverConfig)
+                        .build();
+        Catalog authenticateCatalog = null;
+        try {
+            flussClusterExtension.start();
+            ServerNode coordinatorServerNode =
+                    flussClusterExtension.getCoordinatorServerNode(clientListenerName);
+            String bootstrapServers =
+                    String.format(
+                            "%s:%d", coordinatorServerNode.host(), coordinatorServerNode.port());
+            authenticateCatalog =
+                    new FlinkCatalog(
+                            CATALOG_NAME,
+                            DEFAULT_DB,
+                            bootstrapServers,
+                            Thread.currentThread().getContextClassLoader(),
+                            Collections.emptyMap());
+            Catalog finalAuthenticateCatalog = authenticateCatalog;
+            assertThatThrownBy(finalAuthenticateCatalog::open)
+                    .cause()
+                    .hasMessageContaining("The connection has not completed authentication yet.");
```

Comment:
Missing to configure security configuration is very common, but the exception message is hard to guide users to configure security configurations. Can we improve the message to indicate this like missing to configure "client.security.xx.*" configs.